README.md

```diff
@@ -1,5 +1,7 @@
 # Create React App
 
+[![Build Status](https://travis-ci.org/facebookincubator/create-react-app.svg?branch=master)](https://travis-ci.org/facebookincubator/create-react-app)
```

Comment:
I am not a fan of this approach as it make it harder to maintain the badges, especially when you have more than one :)